### ISSUE TITLE
Does anyone face the same issue?

### ISSUE THREAD

**Issue by saitaiky** [jisungk/deepjazz#12]
def __sample(a, temperature=1.0):
    a = np.log(a) / temperature
    a = np.exp(a) / np.sum(np.exp(a))
    return np.argmax(np.random.multinomial(1, a, 1))

It shows up multinomial has an error
 np.argmax(np.random.multinomial(1, a, 1))
File "mtrand.pyx", line 4593, in mtrand.RandomState.multinomial (numpy\random\mtrand\mtrand.c:37541)
ValueError: sum(pvals[:-1]) > 1.0

**Comment by Echo-Ji** [comment]
Yeah, when i make epochs greater than 26, this error will appear and I  really want to know how to solve it.

**Comment by emc2-2022** [comment]
facing the same problem.

**Comment by cagostino** [comment]
A workaround for this problem from essentially a clone of this repo is available here:
https://github.com/llSourcell/How-to-Generate-Music-Demo/issues/4

**Comment by GyeowoonJeong** [comment]
I found that someone said it occurs because of type casting.
I fixed like this, and I could fix the error.

def __sample(a, temperature=1.0):
    a = np.asarray(a).astype('float64')
    a = np.log(a) / temperature
    a = np.exp(a) / np.sum(np.exp(a))
    return np.argmax(np.random.multinomial(1, a, 1))
